timjwkim/cdl-manager
# Added ability to upload files and added backend connection

Things changed:
- Refactored data/data.js to be more usable when making api call
- Added api calls to retrieve data
- Added button to allow for file uploads

How to test:
`npm run dev`

To check api:
Go to `localhost:3000/api/teams/<teamName>`

To check file upload:
Go to `localhost:3000` and click button to upload a file
